telco-examples/mgmt-cluster/airgap/README.md

```diff
@@ -0,0 +1,183 @@
+
+- `${SCC_ACCOUNT_EMAIL}` - The email address for the SUSE Customer Center account. This could be obtained from the SUSE Customer Center and replacing the value in the `99-register.sh` file.
+
+You need to modify the following folder:
+
+- `base-images` - To include inside the `SLE-Micro.x86_64-5.5.0-Default-SelfInstall-GM2.install.iso` image downloaded from the SUSE Customer Center.
+
+You need to modify the `custom/files/` folder to include the necessary files to be used in the air-gap scenario:
+
+- `airgap-resources.tar.gz` - This file contains the necessary files to be used in the air-gap scenario. You need to prepare the tarball before starting the EIB build process.
+> ```
+>  airgap-resources
+>  |-- clusterctl
+>  |-- clusterctl.yaml
+>  `-- overrides
+>      |-- bootstrap-rke2
+>      |   `-- v0.2.6
+>      |       |-- bootstrap-components.yaml
+>      |       `-- metadata.yaml
+>      |-- cluster-api
+>      |   `-- v1.6.2
+>      |       |-- core-components.yaml
+>      |       `-- metadata.yaml
+>      |-- control-plane-rke2
+>      |   `-- v0.2.6
+>      |       |-- control-plane-components.yaml
+>      |       `-- metadata.yaml
+>      `-- infrastructure-metal3
+>          `-- v1.6.0
+>              |-- cluster-template.yaml
+>              |-- infrastructure-components.yaml
+>              `-- metadata.yaml 
+> ```
+
+where the `clusterctl` is the binary file to be used to create the management cluster, the `clusterctl.yaml` is the configuration file to be used by the `clusterctl` binary, and the `overrides` folder contains the necessary files to be used by the `clusterctl` binary to create the management cluster.
+
+The `clusterctl.yaml` file contains the following content:
+
+```yaml
+providers:
+  # override a pre-defined provider
+  - name: "cluster-api"
+    url: "/root/cluster-api/overrides/cluster-api/v1.6.2/core-components.yaml"
+    type: "CoreProvider"
+  - name: "metal3"
+    url: "/root/cluster-api/overrides/infrastructure-metal3/v1.6.0/infrastructure-components.yaml"
+    type: "InfrastructureProvider"
+  - name: "rke2"
+    url: "/root/cluster-api/overrides/bootstrap-rke2/v0.2.6/bootstrap-components.yaml"
+    type: "BootstrapProvider"
+  - name: "rke2"
+    url: "/root/cluster-api/overrides/control-plane-rke2/v0.2.6/control-plane-components.yaml"
+    type: "ControlPlaneProvider"
+images:
+  all:
+    repository: registry.suse.com/edge
+```
+
+> **_IMPORTANT:_**  
+> if you want to use a different version of the `cluster-api`, `cluster-api-provider-rke2` and `cluster-api-provider-metal3` repositories, you need to modify the curl commands to download the necessary files.
+
+A list of curl commands to download the necessary files to be used in the air-gap scenario is provided:
+
+```bash
+# clusterctl binary
+curl -L https://github.com/kubernetes-sigs/cluster-api/releases/download/1.6.2/clusterctl-linux-${GOARCH} -o /usr/local/bin/clusterct
```

Comment:
Users won't necessarily have GOARCH defined in their environment and we currently only support the EIB step on x86 so perhaps replace `${GOARCH}` with [amd64](https://github.com/kubernetes-sigs/cluster-api/releases/download/v1.6.2/clusterctl-darwin-amd64) ?

Also the -o name is truncated to `clusterct`

Reply:
good catch! I will change it. Thanks